python_modules/dagster/dagster/_model/__init__.py

```diff
@@ -15,3 +17,9 @@ def __init__(self, **data: Any) -> None:
     class Config:
         extra = "forbid"
         frozen = True
+
+    def model_copy(self, *, update: Optional[Dict[str, Any]] = None, deep: bool = False) -> Self:
```

Comment:
if we want to expose `deep` should probably put it under test 

since these are (generally) immutable maybe we don't need to expose deep

Reply:
👍 removed